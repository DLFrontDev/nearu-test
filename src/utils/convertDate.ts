export const convertDate = (date: string | null) =>
  typeof date == "string" ? date.split("-").reverse().join("/") : null;
