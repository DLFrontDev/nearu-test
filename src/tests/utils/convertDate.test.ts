// sum.test.js
import { expect, test } from "vitest";
import { convertDate } from "@/utils/convertDate";

test("Converts date format from string", () => {
  expect(convertDate("2000-01-02")).toMatchSnapshot();
});

test("Converts date format from null", () => {
  expect(convertDate(null)).toBe(null);
});
