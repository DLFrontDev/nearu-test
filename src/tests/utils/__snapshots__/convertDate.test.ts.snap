// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Converts date format from string 1`] = `"02/01/2000"`;
